index.js

```
@@ -83,15 +85,25 @@ const MultiselectPrompt = {
 		this.render()
 	}
 	, right: function () {
-		this.value[this.cursor].selected = true
-		this.render()
+		if (this.value.filter (e => e.selected).length < this.maxChoices) {
```

As below, please adjust the function to fail fast.

---

Sorry @derhuerst I'm not quite sure I'm following how can you make 

```Javascript
if (this.value.filter (e => e.selected).length < this.maxChoices) {
			this.value[this.cursor].selected = true
			this.render()
		}
		else
			return this.bell()
```
fail faster? It checks if the number of selected items is inferior than `maxChoices`, if not it fails `return this.bell()` immediately.
I'm not sure what you'd like :/

---

```js
if (this.value.filter(e => e.selected).length >= this.maxChoices) return this.bell()
this.value[this.cursor].selected = true
this.render()
```

---

Ho okay, this is purely syntactical right?
Because this is the exact same code otherwise. I'll change it, no problem.